prow/cluster/starter/hook.deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: hook
+  labels:
+    app: hook
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
```

Comment:
Same comment as with deck. There are more reasons to do this for hook since its teardown tends to be longer because of the lack of https://github.com/kubernetes/test-infra/issues/3396 and its startup is also going to get slower because we should require cloning of the repositories that use approvals upfront on a new rollout once we move approvals in prow.

Reply:
Now 2 replicas.